src/pages/AppDetail.svelte

```diff
@@ -5,19 +5,46 @@
   import FilterInput from "../components/FilterInput.svelte";
   import AppAlert from "../components/AppAlert.svelte";
   import NotFound from "../components/NotFound.svelte";
+  import Pagination, {
+    makePages,
+    goToPage,
+  } from "../components/Pagination.svelte";
 
   export let params;
   const URL = `data/${params.app}/index.json`;
   let app;
+  let to;
+  let from;
+  let total;
+  let metrics = [];
+  let pages = [];
+  let paginationData;
+  let lastPage;
+  let currentPage = 1;
```

Comment:
I wonder if we could replace most of this with a structure called `paginationState`, and just do the book-keeping/updating of the variables there. That would eliminate the need to pass a bunch of individual variables back to the pagination component.

Reply:
@wlach that has been done, it makes things way cleaner.